lib/lambda_ethereum_consensus/fork_choice/store.ex

```diff
@@ -65,6 +65,7 @@ defmodule LambdaEthereumConsensus.ForkChoice.Store do
     # TODO: this should be done after validation
     :ok = StateStore.store_state(anchor_state)
     :ok = BlockStore.store_block(signed_anchor_block)
+    schedule_next_tick()
```

Comment:
should we maybe ensure that the time is correctly set here before starting the Genser? Otherwise we will have a full second with a wrong time and there could be some undesired side effects.

Reply:
The time is set according to the retrieved finalized state inside `get_forkchoice_store`

Reply:
Correct, but it is the time of the slot in the anchor_block. In checkpoint sync, it might not refer to the current time, but several minutes before.

Reply:
Added the change in f25c895